app/scripts/views/mixins/display-name-mixin.js

```diff
@@ -0,0 +1,20 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// helper functions for view with display name. Meant to be mixed into view.
+
+define(function (require, exports, module) {
+  'use strict';
+
+  module.exports = {
```

Comment:
@divyabiyani hey, thanks for the pull request. You probably do not need to introduce a mixin here because all of the display name logic is already in `settings/display_name.js`. Could we just add it there and add new tests to the existing display_name.js unit test?